invenio.cfg

```diff
@@ -297,6 +302,23 @@ RDM_RECORD_CLS = ZenodoRDMRecord
 RDM_DRAFT_CLS = ZenodoRDMDraft
 RDM_RECORDS_SERIALIZERS = record_serializers
 
+
+block_message = (
+    "Our spam protection system has classified your upload as "
```

Comment:
We don't have a published content policy. Also, we only block when we are quite sure it's spam, so I would not say potentially.

**Your account has been blocked** - the record you tried to publish has been classified as spam by our automated spam protection system. Your account has therefore been blocked and the record has not been published. If you believe this is a mistake, please contact us on support.

Reply:
Changing, though we don't have a lot of flexibility on message formatting at the moment (i.e. no bold/emphasis, etc.).